modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/dataSetsPermissions.spec.ts

```diff
@@ -56,33 +59,11 @@ async function openActionsDropdown({page, text}: {page: Page; text: string}) {
 }
 
 test.beforeEach(async ({page}) => {
-	if (
-		await page
-			.getByRole('button', {
-				name: 'Sign In',
-			})
-			.isVisible()
-	) {
-		await test.step('Sign in as admin', async () => {
-			await performLogin(page, 'test');
-		});
-	}
+	await performLogin(page, 'test');
```

Comment:
Based on @dsanz suggestion, removing the `login` fixture and manually login/change user works as expected.

Reply:
This is much nicer, a consistent behavior rather than patch of errors :+1: 

Out of curiosity, why doesn't login fixure work correctly?

Reply:
Nobody who I've asked around knows for sure. Me neither.

According to [PW execution order docs](https://playwright.dev/docs/test-fixtures#execution-order), the automatic fixture runs first, for all tests. This shall guarantee a logged in user all the time.

I suspect login fixture kind of runs differently, perhaps on a separate "thread" for some reason. I do suspect on it being a function and thus being explicitly called in `mergeTests()`, as opposed to the regular fixtures, which likely call to `test.extend()` earlier because they do export the result of such `extend()` call for import by tests. In this sense, login fixture would work more like a sort of dynamic import, right before `mergeTest()` puts them together.

This might interfere with normal fixture setup managed by PW (lazily set up right before using by a test, or `auto`matically run before tests). It may also be just a race condition in case before hook /test attempts to log out "too quickly".

So, no idea of what's going on, tbh

Still as long as @juanjofgliferay is confident about observed behavior, I have no concern about merging this